sklearn/utils/estimator_checks.py

```diff
@@ -269,6 +271,40 @@ def _yield_all_checks(name, estimator):
     yield check_fit_idempotent
 
 
+def set_check_estimator_ids(val):
```

Comment:
Should this be linked in  the API reference? We mention it in the contributing guide but it's not documented anywhere.